app/routes/items.js

```diff
@@ -0,0 +1,61 @@
+var log = require('../lib/log.js'),
+ Item = require('../models/item');
+
+/**
+ * Strider API - Items Router
+ * @param {Object} Strider Strider main object
+ */
+
+module.exports = function(Strider) {
+
+  Strider.app.get('/items', function (req, res, next) {
+      Item.find(function(err, items) {
+        if(err) {
+          res.send(err);
+          return next();
+        }
+
+        res.send(items);
+        return next();
+      });
+  });
+
+  Strider.app.post('/items', function (req, res, next) {
+    var item = new Item(req.body);
+    
+    console.log('About to save');
```

Comment:
Non dev debug messages should use winston: log.debug(...) (see the first require) so that they are properly logged.

console.log() should just be removed when pushing then :)